doc/setting_up_sidewalk_environment/setting_up_sidewalk_product.rst

```diff
@@ -42,27 +46,70 @@ Generating provisioning
 
 #. Run device provisioning scripts:
 
-   .. code-block:: console
+   a. Setup Python virtual environment for the provisioning tools
+
+      .. code-block:: console
+
+         $ python3 -m pip install --user virtualenv
+         $ python3 -m venv sample-app-env
+         $ source sample-app-env/bin/activate
+         (sample-app-env)$ python3 -m pip install --upgrade pip
+         (sample-app-env)$ python3 -m pip install -r requirements.txt
+
+   #. Run device provisioning scripts:
+
+      .. code-block:: console
+   
+         (sample-app-env)$ python3 EdgeDeviceProvisioning/provision_sidewalk_end_device.py
```

Comment:
```suggestion
         python3 EdgeDeviceProvisioning/provision_sidewalk_end_device.py
```

Reply:
I would like to mark, this commands must be run in virtual env